scss/components/_tabs.scss

```diff
@@ -138,7 +138,7 @@ $tab-content-padding: 1rem !default;
   display: none;
   padding: $padding;
 
-  &[aria-hidden="false"] {
+  &[aria-hidden=""] {
```

Comment:
@DaSchTour 
No bro .... if i remove .... nothings works .... all tabs content get hidden both on click and by default
Can you pull it once and look into it??